pypeit/core/flexure.py

```diff
@@ -289,7 +291,7 @@ def spec_flex_shift(obj_skyspec, arx_skyspec, arx_lines, mxshft=20):
 
     return dict(polyfit=fit, shift=shift, subpix=subpix_grid,
                 corr=corr[subpix_grid.astype(np.int)], sky_spec=obj_skyspec, arx_spec=arx_skyspec,
-                corr_cen=corr.size/2, smooth=smooth_sig_pix, success=success)
+                corr_cen=corr.size/2, smooth=smooth_sig_pix)
```

Comment:
How were the calling functions handling the case when `success = False`?  Do they now know how to handle if the returned value is `None`?

Reply:
From looking at this code, I found that the calling function `spec_flexure_slit` doesn't actually use the `success` info at all. This is why I removed `success` from the dict. It deals instead with `None`. So if the returned value is `None` the flexure correction is not performed, which (in my opinion) is much better than getting a 5000 pixels shift in the wavelength direction.

Reply:
Agreed!!